nirc_ehr/resources/schemas/dbscripts/postgresql/nirc_ehr-21.023-22.000.sql

```diff
@@ -0,0 +1,78 @@
+CREATE TABLE nirc_ehr.Staff
```

Comment:
I don't see an edit to NIRC_EHRModule.java to bump the schema version. You probably need to go to 22.001 with the back port of the schema version bump from develop.

Reply:
Right, and script probably needs to be named `nirc_ehr-22.000-22.001.sql` to run on all dev/test machines.

Reply:
Fixed this. Rushed out the PR before I tested the startup script.